profiles/base.yaml

```diff
@@ -382,6 +382,7 @@ storage_containers:
 gear:
 gear_sets:
 empath_healing:
+saferoom_health_threshold: 25
```

Comment:
Wouldn't be bad, I'd like if there was an override where a bleeder always goes for a heal. The two situations that worry me are nerve damage spirals not being stopped soon enough and not healing bleeders.